test/test_type_info.py

```diff
@@ -14,17 +14,17 @@
 class TestDTypeInfo(TestCase):
 
     def test_invalid_input(self):
-        for dtype in [torch.float32, torch.float64]:
+        for dtype in [torch.float16, torch.float32, torch.float64]:
             with self.assertRaises(TypeError):
                 _ = torch.iinfo(dtype)
 
-        for dtype in [torch.int64, torch.int32, torch.int16, torch.uint8]:
+        for dtype in [torch.int64, torch.int32, torch.int16, torch.int8, torch.uint8]:
             with self.assertRaises(TypeError):
                 _ = torch.finfo(dtype)
 
     @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
     def test_iinfo(self):
-        for dtype in [torch.int64, torch.int32, torch.int16, torch.uint8]:
+        for dtype in [torch.int64, torch.int32, torch.int16, torch.int8, torch.uint8]:
```

Comment:
Can this also test that xinfo.dtype is equal to xninfo.dtype?

Reply:
Addressed!

Reply:
Thank you!